Serverless SQL Database: Update known-differences.mdx (#3762)
scaleway/docs-content: serverless/sql-databases/reference-content/known-differences.mdx

    NOTIFY channel, 'message';
    ```

> - Creating temporary tables or views cannot be performed.

Reviewer: ```suggestion
- It is not possible to create temporary tables or views.
```

    You can still refresh materialized views without the `CONCURRENTLY` option.
    </Message>

> - Declaring a cursor in transactions with `WITH HOLD` can be performed, but accurate cursor handling is not guaranteed, especially when using multiple connections.

Reviewer: ```suggestion
- You can declare a cursor in transactions with `WITH HOLD`, but accurate cursor handling is not guaranteed, particularly when using multiple connections.
```

    CREATE SUBSCRIPTION subscription CONNECTION 'connection' PUBLICATION publication;
    ```

> - Preparing transactions cannot be performed.

Reviewer: ```suggestion
- It is not possible to prepare transactions.
```

    SET TIME ZONE 'Europe/Paris';
    RESET timezone;
    ```

>     If you require this features to work consistently for a given SQL query, we suggest you set them in transactions:

Reviewer: ```suggestion
    If you require these features to work consistently for a given SQL query, we suggest you set them in transactions:
```
